Samples/VirtualCamera/README.md

```diff
@@ -0,0 +1,93 @@
+# Virtual Camera project
+
+This is a sample on how to create a virtual camera on windows. This project is consists of 5 parts
+1. VirtualCameraWin32 <br>
+MediaSource for the virtual camera. 
+
+2. VirtualCamera_ConsoleApp <br>
+Application use to create virtual camera on the system.
+
+3. VirtualCamera_AppMSI <br>
+Installable application to deploy and remove media source from the system
+
+4. VirtualCamera_AppMSIX <br>
+Package win32 console appliaction in to UWP

```

Comment:
>in to UWP [](start = 34, length = 9)

Let us not mix up UWP and MSIX. Let us use "Windows Store deployable package" instead.

Reply:
Sync up with Louie, he will be checking in the UWP sample after this complete